docker/docker-compose.yml

```diff
@@ -89,87 +103,95 @@ services:
       "--useDevPrivilegeKeys=true",
       "--nodePort=3002",
       "--appName=haveno-XMR_LOCAL_Seed2_3002",
-      "--xmrNode=http://localhost:28081"
+      "--xmrNode=http://node2:28081"
     ]
-    network_mode: "host"
     depends_on:
       - node2
+    ports:
+      - "127.0.0.1:3002:3002"
 
-  arbitrator:
-    build: haveno/
-    container_name: arbitrator
-    entrypoint: [ "./haveno-daemon" ]
-    command: [
-      "--baseCurrencyNetwork=XMR_LOCAL",
-      "--useLocalhostForP2P=true",
-      "--useDevPrivilegeKeys=true",
-      "--nodePort=4444",
-      "--appName=haveno-XMR_LOCAL_arbitrator",
-      "--apiPassword=apitest",
-      "--apiPort=9998",
-      "--walletRpcBindPort=38090",
-      "--passwordRequired=false",
-    ]
-    network_mode: "host"
-    depends_on:
-      - seed1
+  # arbitrator:
```

Comment:
I saw that the log files were not collected for the arbitrator, user1, and user2. Guessing because these were commented out. Should be uncommented?

Reply:
No, because these containers are currently unused. Tests are invoking arbitrator, user1 and user2 from haveno directory. To fully dockerize the project we'd need to invoke monerod and haveno processes with `docker run`.

If I'd uncomment these containers, there would be binding errors since docker and local haveno will be racing to bind to the same port. If I remove the port exposure from docker, then logs will be essentially empty (as in current master).

Reply:
How can we get the logging for arbitrator, user1, and user2 to remain collected in the CI tests then? Those logs are sometimes needed for debugging.

Reply:
On master, these logs are populated:

<img width="365" alt="image" src="https://github.com/user-attachments/assets/a7c185b8-5d19-4caa-a941-2999860f42c5">